### DISCUSSION TITLE
Don't use p-value for GO-MWU if you have RNA-seq data!

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1342] [Show and tell]
While writing up my results, I noticed a change in the official [GO-MWU README file](https://github.com/z0on/GO_MWU/blob/master/README.md). In June, they added the following section:

`NOTE: In read-based gene expression analysis (RNA-seq, TagSeq) p-values may be biased towards highly abundant genes, especially when the read depth is low. This may result in the corresponding GO bias. Use log2-fold changes to avoid this.`

I switched all my GO-MWU analyses from unadjusted p-value to log2-fold change as a result, and saw some quite substantial changes in the output. So if you ran GO-MWU on RNA-seq data a while back, you definitely might want to redo those analyses!

**Comment by AHuffmyer** [comment]
Thanks for posting this!! I'm guessing this doesn't affect using module membership scores from WGCNA data?

**Reply by aspencoyle** [reply]
Nope, I think no changes in that case!
